Documentation/.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
Shouldn't this be moved to the global .gitignore file?

Reply:
No, as we use this folder for development in `composer.json`.
Everyone who will work on this extension and runs `composer install` will have this folder which should be ignored. So it's a project folder.

Reply:
Sorry i meant the .gitignore file in the root dir of the extension.

This .gitignore file is in the Documentation folder.

Reply:
Fixed